pkg/kn/config/config.go

```diff
@@ -176,6 +199,22 @@ func initDefaults() *defaultConfig {
 		configFile:          defaultConfigLocation("config.yaml"),
 		pluginsDir:          defaultConfigLocation("plugins"),
 		lookupPluginsInPath: true,
+		sinkMappings: []SinkMapping{
+			{
+				Prefix:   "svc",
+				Resource: "services",
+				Group:    "core",
+				Version:  "v1",
+			},
+		},
+		channelTypeMappings: []ChannelTypeMapping{
+			{
+				Alias:   "Kafka",
+				Group:   "messaging.knative.dev",
+				Version: "v1alpha1",
+				Kind:    "KafkaChannel",
+			},
+		},
```

Comment:
Since the last changes and example config being created those definition can be removed from default config.

Reply:
DONE